ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/disconnected/secureboot/cucushift-installer-rehearse-vsphere-upi-disconnected-secureboot-workflow.yaml

```diff
@@ -2,19 +2,9 @@ workflow:
   as: cucushift-installer-rehearse-vsphere-upi-disconnected-secureboot
   steps:
     pre:
-    - ref: ipi-install-rbac
-    - ref: openshift-cluster-bot-rbac
-    - ref: ipi-conf
-    - ref: ipi-conf-vsphere-check
-    - ref: ipi-conf-vsphere-mirror-image
-    - ref: upi-conf-vsphere
-    - ref: upi-conf-vsphere-ova
-    - ref: upi-install-vsphere
-    - ref: ipi-install-times-collection
+    - chain: cucushift-installer-rehearse-vsphere-upi-disconnected-provision
```

Comment:
To distinguish  cucushift-installer-rehearse-vsphere-upi-disconnected workflow, I suggest to create a dedicated provision chain for this workflow, and move `SECURE_BOOT_ENABLED` into that chain. Some similar things like https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/proxy/provision/cucushift-installer-rehearse-azure-ipi-proxy-provision-chain.yaml#L17

Reply:
get it, updated.